packages/ownable/src/lib.rs

```diff
@@ -79,7 +258,8 @@ pub enum OwnershipError {
 }
 
 /// Storage constant for the contract's ownership
-const OWNERSHIP: Item<Ownership<Addr>> = Item::new("ownership");
+pub const OWNERSHIP_KEY: &str = "ownership";
+pub const OWNERSHIP: OwnershipStore = OwnershipStore::new(OWNERSHIP_KEY);
```

Comment:
Think this should stay private. The use of the lib is oriented around the exposed fns (`cw_ownable::xyz()`) similar to cw2, let's keep it that way.

By exposing the underlying structure more advanced users can still create custom stores.

Reply:
`OWNERSHIP_KEY` can be made private, but `OWNERSHIP` should be public, so 3rd parties can use it for multiple ownerships. Like here: https://github.com/public-awesome/cw-nfts/blob/minor_changes/packages/cw721/src/state.rs#L6

Reply:
They can just use the default exposed functions no?

Reply:
Well, I could do so, but in this case I need to import all functions like `get_ownership`, `update_ownership`, `initialize_owner`, etc.

Code doesnt look nice then:

```rs
            MINTER.initialize_owner(storage, api, Some(legacy_minter.as_str()))?;
            initialize_owner(storage, api, Some(legacy_minter.as_str()))?;
```

instead of:

```rs
            MINTER.initialize_owner(storage, api, Some(legacy_minter.as_str()))?;
            CREATOR.initialize_owner(storage, api, Some(legacy_minter.as_str()))?;
```

Reply:
You could define your own version of the `CREATOR`, just like the `MINTER`. Should be easy if we expose the key. Then you can call it whatever you want as well.